test/AutoRest.TestServerLowLevel.Tests/dpg-customization.cs

```diff
@@ -60,6 +60,17 @@ await foreach (Page<BinaryData> page in allPages.AsPages())
             }
         });
 
+        [Test]
+        public Task HandwrittenModelLro() => Test(async (host) =>
+        {
+            using var diagnosticListener = new ClientDiagnosticListener("dpg_customization_LowLevel", asyncLocal: true);
+            CollectionAssert.IsEmpty(diagnosticListener.Scopes);
+
+            var lro = await new DPGClient(Key, host).LroValueAsync(WaitUntil.Started, "model");
+            await lro.WaitForCompletionAsync();
+            Assert.AreEqual("model", $"{lro.Value.Received}");
```

Comment:
Can we check if any scope created after calling `LroValueAsyn` using diagnosticListener?

Reply:
Sure!

Reply:
Done